### PR TITLE
Add support page

### PR DESCRIPTION
**Pull request by devongovett**

### CODE REVIEW

File: packages/dev/docs/pages/Support.mdx [hunk 1]
```diff
@@ -12,10 +12,56 @@ export default Layout;
...
+share these running examples so that the core team and the community can play around with your scenario and better understand
+your question.
+
+We try our best to reply to questions as quickly as possible, but also rely on help from the community. If you see a question
+and know the answer, please don't hesitate to reply yourself. This helps reduce the number of questions the core team needs
+to answer and improves response time. Thanks in advance for your help!
+
+## Reporting issues
+
+Please see our [contributing](contributing.html#reporting-issues) guide for information about how to report bugs or security issues, and request
```
Comment (dannify) [inline packages/dev/docs/pages/Support.mdx:37]:
```suggestion
Please see our [contributing](contribute.html#reporting-issues) guide for information about how to report bugs or security issues, and request
```

File: packages/dev/docs/pages/Support.mdx [hunk 2]
```diff
@@ -12,10 +12,56 @@ export default Layout;
...
+assigned to a team member, then it is in progress. If it's been placed in a milestone, then you can expect it to be released
+in that version.
+
+### Prioritization
+
+If you would like to help expedite a feature, there are a few ways to help. In prioritizing our work, the core team looks at community
+demand for a feature. Using the 👍 emoji reaction on issues helps us know how popular a request is. Please use it to indicate to
+us that you're interested in a feature.
+
+The fastest way to get a feature implemented is to to contribute it yourself! Please read our [contributing guide](contributing.html)
```
Comment (dannify) [inline packages/dev/docs/pages/Support.mdx:58]:
```suggestion
The fastest way to get a feature implemented is to to contribute it yourself! Please read our [contributing guide](contribute.html)
```

File: packages/dev/docs/pages/Support.mdx [hunk 3]
```diff
@@ -12,10 +12,56 @@ export default Layout;
...
+for more information. Please post a comment on the issue indicating that you'd like to work on it. A core team member will reach
+out to discuss how we'd like to go about it. This helps avoid rework in the PR review process.
+
+### Release schedule
+
+We generally plan to release every two weeks. However, priority bug fixes may be released more quickly. In addition, we release
+alpha and beta versions of new components and features we're working on in advance of their stable release so that the community
+can help test them out. Check out the [Versioning](react-spectrum/Versioning.html) docs to learn more about how we version and
```
Comment (dannify) [inline packages/dev/docs/pages/Support.mdx:66]:
```suggestion
can help test them out. Check out the [Versioning](react-spectrum/versioning.html) docs to learn more about how we version and
```

File: packages/dev/docs/pages/Support.mdx [hunk 4]
```diff
@@ -12,10 +12,56 @@ export default Layout;
...
 category: Foundation
-description: How to contribute features, report bugs, or make suggestions to improve React Spectrum.
+description: How to get help with React Spectrum, how to report issues, and our development roadmap.
 keywords: [react spectrum support, react spectrum help]
 ---
 
 # Support
 
-For feedback, bug reports, general questions about React Spectrum, please visit https://github.com/adobe/react-spectrum/issues/.
+This page includes information about how to get help with React Spectrum, how to report issues, and our development roadmap.
```
Comment (snowystinger) [inline packages/dev/docs/pages/Support.mdx:21]: This page includes getting help with React Spectrum, reporting issues, and information on our development roadmap.

File: packages/dev/docs/pages/Support.mdx [hunk 5]
```diff
@@ -12,10 +12,56 @@ export default Layout;
...
+
+## Getting help
+
+The best place to ask questions about React Spectrum and get help is on Github. Please [file an issue](https://github.com/adobe/react-spectrum/issues)
```
Comment (snowystinger) [inline packages/dev/docs/pages/Support.mdx:25]: The best place to ask questions and get help about React Spectrum is on Github. Please [file an issue](https://github.com/adobe/react-spectrum/issues)

File: packages/dev/docs/pages/Support.mdx (hunk 4 above)
Comment (devongovett) [reply packages/dev/docs/pages/Support.mdx:21]: "This page includes getting help" doesn't make sense to me. Feels like it needs the "information about"

**Pull request merged**